Cookbook/CookbookCommon/Sources/CookbookCommon/Recipes/MiniApps/Telephone.swift

```diff
@@ -230,6 +232,12 @@ struct Phone: View {
         }
         .padding()
     }
+
+    // MARK: - Phone - Private
+
+    private func formattedPhoneNumber(_ digits: String) -> String {
+        return digits == "" ? " " : digits
```

Comment:
With an empty string, the VStack collapses since the Text's vertical space is not retained, this ensures that doesn't happen (incidentally, is also why I expect `last10Digits` defaulted to " ").